.travis.yml

```diff
@@ -11,6 +11,7 @@ language: java
 install: true
 jdk:
  - oraclejdk8
+ - openjdk10
```

Comment:
I'm not sure why we need to install `openjdk10` in the travis configuration for this PR. If you simply wanted to test, then can we remove it for merging?